modules/3d-tiles/src/tz3-loader.ts

```diff
@@ -0,0 +1,41 @@
+import {LoaderOptions, LoaderWithParser} from '@loaders.gl/loader-utils';
+import {DataViewFile} from '@loaders.gl/zip';
+import {parse3tz as parse3tzFromProvider} from './tz3/tz3-parser';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+/** options to load data from 3tz */
+export type TZ3LoaderOptions = LoaderOptions & {
+  tz3?: {
+    /** path inside the 3tz archive */
+    path?: string;
+  };
+};
+
+/**
+ * Loader for 3tz packages
+ */
+export const TZ3Loader: LoaderWithParser<Buffer, never, TZ3LoaderOptions> = {
```

Comment:
Nit: Are we better off calling this `Tiles3DArchiveLoader`? 

I suspect it will make this loader more easily understood by users who have never heard about `3tz` and even more confusingly, we have to reorder the letters to `tz3`...

3tz feels more like the file extension. https://github.com/erikdahlstrom/3tz-specification

Reply:
Also since we only load a single file, maybe call it `Tiles3DArchiveFileLoader`. Had a similar discussion with @belom88 around the Zip loader.